src/components/manifold-plan-table/metered-feature.tsx

```diff
@@ -20,28 +21,36 @@ const MeteredFeature: FunctionalComponent<MeteredFeatureProps> = ({ feature }) =
   }
 
   const { costTiers = [], unit } = numericDetails;
+  const multitiered = costTiers.length > 1;
 
   return (
     <div class="mp--cell mp--cell__body mp--cell__block">
       <div class="mp--metered">
         <div class="mp--metered__header">
-          <p class="mp--metered__header-text">cost</p>
-          <p class="mp--metered__header-text mp--metered__header-tar">usage range</p>
+          <p class="mp--metered__header-text">billed per {singularize(unit)}</p>
```

Comment:
Just checking.. did we remove the "Cost" heading on the tier table? Your screenshot has it, but the code seems to suggest we did decide to remove it (which I agree with)

Reply:
Oh yeah I didn’t update the screenshot in the PR. The one from Slack was more recent

Reply:
![Screen Shot 2020-03-27 at 11 28 37](https://user-images.githubusercontent.com/1369770/77920495-c7544300-725b-11ea-8403-bb779947be94.png)

![Screen Shot 2020-03-27 at 11 34 03](https://user-images.githubusercontent.com/1369770/77920510-ca4f3380-725b-11ea-9ec0-92b9af43b9eb.png)

Reply:
Yeah I like this